### PR TITLE
ADEN-8215 MEGA everywhere in Oasis

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/remnantGpt.js [hunk 1]
```diff
@@ -33,7 +32,7 @@ define('ext.wikia.adEngine.provider.remnantGpt', [
 				slotTweaker.removeDefaultHeight(slotName);
 			},
 			getAdUnitBuilder: function () {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/remnantGpt.js]: Not needed

File: extensions/wikia/AdEngine/js/provider/remnantGptMobile.js [hunk 2]
```diff
@@ -28,7 +27,7 @@ define('ext.wikia.adEngine.provider.remnantGptMobile', [
 		},
 		{
 			getAdUnitBuilder: function () {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/remnantGptMobile.js]: Not needed

File: extensions/wikia/AdEngine/js/video/player/porvata/googleImaSetup.js [hunk 3]
```diff
@@ -31,9 +26,8 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaSetup', [
 		var vastUrlBuilderOptions = {},
 			vastUrl;
 
-		if (megaIsEnabled(params)) {
-			vastUrlBuilderOptions.adUnit =
-				megaAdUnitBuilder.build(getPosBasedOnProduct(params), params.vastTargeting.src);
+		if (params.vastTargeting && params.vastTargeting.src && (params.adProduct || params.vastTargeting.pos)) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/video/player/porvata/googleImaSetup.js:29]: Do we need this if statement? How did it work before?

Comment (nandy-andy) [reply extensions/wikia/AdEngine/js/video/player/porvata/googleImaSetup.js:29]: Porvata2 will disappear, so I didn't want to dive more into it. Unit tests pass with it ;)

**Review by fraszczakszymon — APPROVED** [review]

**Review by dmnsobczak — APPROVED** [review]

**Pull request merged**